templates/dotnet/publish-nuget-packages.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  - name: nugetProjectPath
+    type: string
+
+  - name: vstsFeed
+    type: string
+    default: 'Video Hearings/vh-packages'
```

Comment:
Can we remove defaults referring to project specific things please

Reply:
Done.